apiv2/internal/server/host.go

```diff
@@ -719,54 +706,84 @@ func (is *InventorygRPCServer) GetHostsSummary(
 	var errorState uint32
 	var runningState uint32
 	var unallocatedState uint32
+	reqFilter := req.GetFilter()
+
+	filterIsFailedHostStatus := `%s = %s OR %s = %s OR %s = %s OR %s.%s = %s OR %s.%s = %s OR %s.%s = %s OR %s.%s = %s`
```

Comment:
Do we need to add an `has(instance)`? I'm not sure how it behaves if we try to filter on the `instance.[field]`, but the instance is not connected.

Reply:
It does not need.